addon/validators/slug.js

```diff
@@ -0,0 +1,16 @@
+import {
+  validatePresence,
+  validateLength,
+  validateFormat
+} from "ember-changeset-validations/validators";
+
+const validateSlug = () => (...args) =>
```

Comment:
Why the extra closure?

Reply:
Because validation functions are higher order functions which return the actual function. See https://github.com/poteto/ember-changeset-validations#synchronous-validators

Reply:
Ah, right!